packages/teams-js/.eslintrc.cjs

```diff
@@ -13,5 +13,6 @@ module.exports = {
     ],
     'no-inner-declarations': 'off',
     'strict-null-checks/all': 'warn',
+    'recommend-no-namespaces/recommend-no-namespaces': 'warn',
```

Comment:
I know there weren't a lot of items before you got here but please consider alphabetizing this.